docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -781,6 +781,118 @@ To read and write a value of the Bluetooth descriptor:
       - write about setWriteValueRequestCallback() for descriptor
  -->
 ### Responding to read and write value requests from server clients
+
+In order to react and respond to the read and write requsts from a client connected to the server running on the device, the callback functions needs to be set on the characteristics or on the descriptors.
+
+To set a callback function for read or write request on a characteristic or descriptor:
+
+1. Prepare a GATT service containing at least one characteristic or a characteristic with at least one descriptor, that can be read or write:
+   ```
+   var exampleDescriptor = {
+      uuid: "0155",
+      /* Descriptor's permissions */
+      readPermission: true,
+      writePermission: true
+   };
+
+   var exampleCharacteristic = {
+      uuid: "0180",
+      descriptors: [exampleDescriptor],
+      /* Characteristic's properties */
+      isReadable: true,
+      isWritable: true,
+      /* Characteristic's permissions */
+      readPermission: true,
+      writePermission: true
+   };
+
+   var gattService = {
+      serviceUuid: "0955",
+      isPrimary: false,
+      services: [],
+      characteristics: [exampleCharacteristic]
+   };
+   ```
+
+2. Register the service:
+   ```
+   var registerServiceSuccesCB = function() {
+      console.log("Service successfully registered!");
+   };
+
+   var registerServiceErrorCB = function(e) {
+      console.log("Service not registered, error: " + e);
+   };
+
+   var server = tizen.bluetooth.getGATTServer();
+
+   server.registerService(gattService, succesCB, errorCB);
+   ```
+
+3. Start the GATT server:
+   ```
+   var serverStartSuccessCB = function() {
+      console.log("Server started successfully!");
+   };
+
+   var serverStartErrorCB = function(e) {
+      console.error("Server didn't start, error: " + e);
+   };
+
+   server.start(serverStartSuccessCB, serverStartErrorCB);
+   ```
+
+4. To register the callback called when a client reads the value of the characteristic from the local GATT server, create the callback function and pass it as an argument to the `setReadValueRequestCallback()` method called on the characteristic object.
+   ```
+   var characteristicReadRequestCallback = function(clientAddress, offset) {
+      console.log(clientAddress + " requested to read characteristic's value with offset: " + offset);
+      return new tizen.GATTRequestReply(0, "0x1234");
+   };
+   server.services[0].characteristics[0].setReadValueRequestCallback(characteristicReadRequestCallback);
```

Comment:
This function takes 5 arguments:

```
void setReadValueRequestCallback(ReadValueRequestCallback readValueRequestCallback, optional SuccessCallback? successCallback,
                                 optional ErrorCallback? errorCallback, optional SuccessCallback? sendResponseSuccessCallback,
                                 optional ErrorCallback? sendResponseErrorCallback);
```

Please, define all of them to show the guide readers how to use this function properly.

Reply:
I have added paragraph with all the callbacks. I made them common for all four functions in this guide section. Please review this change.